images/driver-base/Dockerfile

```diff
@@ -15,13 +15,6 @@
 FROM photon:3.0
 LABEL "maintainers"="Divyen Patel <divyenp@vmware.com>, Sandeep Pissay Srinivasa Rao <ssrinivas@vmware.com>, Xing Yang <yangxi@vmware.com>"
 
-RUN tdnf -y remove toybox
```

Comment:
I thought you wanted to remove toybox, why are you retaining it in the image?

Reply:
removal of toybox is installing gmp, grep, and coreutils packages and adding up more space rather than helping to reduce docker image size. so retained the toybox util.

```
RUN tdnf -y remove toybox
 ---> Running in 190528e5510e
Refreshing metadata for: 'VMware Photon Extras 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64) Updates'

Installing:
gmp                      x86_64       6.1.2-3.ph3      photon-updates 524.08k 536656
grep                     x86_64       3.1-3.ph3        photon-updates 241.50k 247301
coreutils                x86_64       8.30-3.ph3       photon-updates   5.84M 6127774

Total installed size:   6.59M 6911731

Removing:
toybox                   x86_64       0.8.2-4.ph3      @System      380.08k 389206

Total installed size: 380.08k 389206
```